panic on resuming check:data v0.4.0

**Describe the bug**
rosetta-cli v0.4.0 panic on resuming check:data with an existing data directory:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x44f1b12]

goroutine 68 [running]:
github.com/coinbase/rosetta-cli/internal/tester.(*DataTester).StartReconciler(0x0, 0x488dec0, 0xc000020ec0, 0x0, 0x0)
	/Users/yutong/code/mygo/rosetta-cli/internal/tester/data.go:272 +0x22
github.com/coinbase/rosetta-cli/cmd.runCheckDataCmd.func2(0x0, 0x0)
	/Users/yutong/code/mygo/rosetta-cli/cmd/check_data.go:134 +0x3f
golang.org/x/sync/errgroup.(*Group).Go.func1(0xc0084b3e60, 0xc0084a38a0)
	/Users/yutong/code/go/pkg/mod/golang.org/x/sync@v0.0.0-20200317015054-43a5402ce75a/errgroup/errgroup.go:57 +0x59
created by golang.org/x/sync/errgroup.(*Group).Go
	/Users/yutong/code/go/pkg/mod/golang.org/x/sync@v0.0.0-20200317015054-43a5402ce75a/errgroup/errgroup.go:54 +0x66
```

**To Reproduce**
Steps to reproduce the behavior:
- start check:data with none empty `data_directory`, `bootstrap_balances` and `interesting_accounts` in config file (not sure if `bootstrap_balances` and `interesting_accounts` matter here)
- cancel the execution
- then resume with same command

**Expected behavior**
no panic

--- a/internal/tester/data.go
+++ b/internal/tester/data.go
@@ -219,7 +219,6 @@ func InitializeData(
 				}
 			} else {
 				log.Println("Skipping balance bootstrapping because already started syncing")
-				return nil
 			}
 		}
 

